dnf/cli/commands/__init__.py

```diff
@@ -248,7 +248,7 @@ class CheckUpdateCommand(Command):
     check-update command.
     """
 
-    aliases = ('check-update', 'check-upgrade')
+    aliases = ('check-update', 'check-upgrade', 'ch-up', 'chup',)
```

Comment:
``ch-up`` is probably better because there is always ``-`` in command

Reply:
Yes, I agree here.